runtime/src/bank_forks.rs

```diff
@@ -158,10 +167,19 @@ impl BankForks {
 
         // Iterate through the heads of all the different forks
         for bank in initial_forks {
-            banks.insert(bank.slot(), bank.clone());
+            banks.insert(
```

Comment:
I see we did it before as well, but any reason we want/need to not use the `BankForks::insert()` fn here and directly do it?

Reply:
yeah, i think this is odd coding. but `Self` (ie `BankFork`) isn't created at the very end.

also, `BankForks::insert()` will be unusable here once after the upcoming scheduler pr lands. that's because `::insert()` will take a parameter for banking/replaying scheduling mode selection..